flybywiresim/a32nx
# feat: EFB display the list of ATC based on the current flight phase

## Summary of Changes
Extends the ATC page (v0.7) in the EFB to filter ATC list based on current flight phase

## Screenshots (if necessary)
![image](https://user-images.githubusercontent.com/8239933/122360331-9d696800-cf56-11eb-960d-6f443c41f935.png)


## References
n/a

## Additional context



ZeroKaa

## Testing instructions
- File a flight plan in Simbrief
- Import your flight plan in EFB
- Do your flight
- Check on each phase (preflight, ground, takeoff, climb, cruise, descend, approach, go-around, done) if the list of controllers proposed in the ATC page is correct.
- You should see the controller for your current phase and the one for the next phase of your flight. For example of take off : tower and departure (if no departure available than radar, etc...)


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
